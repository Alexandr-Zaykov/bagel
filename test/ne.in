Molecule: C1
  Atom(Ne,  0.0, 0.0, 0.0)

Basis:
  cc-pVDZ cc-pVDZ-F12-CABS

Periodic:
  K = 0
  Period 5.0

MP2:
  frozen

