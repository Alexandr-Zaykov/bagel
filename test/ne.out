
  --------------------------------------------------
  This is the package       ** Poly-R12 **          
  written by Toru Shiozaki (shiozaki.toru@gmail.com)
  --------------------------------------------------

  === Geometry ===

  Symmetry: c1

  Ne   0.0000000000   0.0000000000   0.0000000000

  * Performs crystalline orbital calculation (1D)

      Parameters : K = 0  (L = 0 and S = 0)
      Cell size  :    5.0000000000

  Orthogonalizing overlap matrix

  Maximum residual in orthogonalization: 1.704605362079343
  Minimum residual in orthogonalization: 0.193304862828609

  Using 307.3 KB hard disk for storing "ERI OBS"
  === Periodic RHF iteration (cc-pVDZ)===
  
      0   -115.3992778531      0.1713972466           0.00
      1   -125.5323186147      0.0627563591           0.00
      2   -127.5911463768      0.0281863073           0.00
      3   -128.1692889441      0.0136086382           0.00
      4   -128.3617759893      0.0067756234           0.00
      5   -128.4340396166      0.0034329987           0.00
      6   -128.4638536525      0.0017628442           0.00
      7   -128.4770362641      0.0009169012           0.00
      8   -128.4892400343      0.0000931751           0.00
      9   -128.4889476402      0.0000555586           0.00
     10   -128.4887391676      0.0000085643           0.00
     11   -128.4887699821      0.0000050747           0.00
     12   -128.4887861076      0.0000001410           0.00
     13   -128.4887784885      0.0000000625           0.00
     14   -128.4887748414      0.0000000081           0.00
     15   -128.4887753588      0.0000000024           0.00
     16   -128.4887755606      0.0000000001           0.00
     17   -128.4887755513      0.0000000000           0.00
     18   -128.4887755518      0.0000000000           0.00
  
    * SCF iteration converged.

      K            0
          -32.765635
           -1.918798
           -0.832097
           -0.832097
   HOCO    -0.832097
   LUCO     1.694558
            1.694558
            1.694558
            2.159425
            5.196711
            5.196711
            5.196711
            5.196711
            5.196711

  === Periodic MP2 calculation ===

    >> Gamma = 1.500000 is used in the MP2-R12 calculation.

  Orthogonalizing overlap matrix

  Maximum residual in orthogonalization: 3.797540510454011
  Minimum residual in orthogonalization: 0.001074712569170

  Creating ERI (pp/ii)  of size 50.2 KB
  done

  Now computing mp2 energy

  MP2 energy: -0.1855232812

  === Periodic MP2-R12 calculation ===

  Using 614.7 KB hard disk for storing "Slater and Yukawa ints"

  Creating Yukawa (ii/ii)  of size 4.1 KB
  done

  Creating Slater (pp/ii)  of size 50.2 KB
  done

  Creating v^ia'_ii, OBS part  of size 84.5 KB
  done

  Creating v^ia'_ii, auxiliary functions  of size 84.5 KB
  done

  Creating F^ia'_ii, OBS part  of size 84.5 KB
  done

  Creating F^ia'_ii, auxiliary functions  of size 84.5 KB
  done

  Entering F * v (ii/ii) OBS contraction...
  done

  Entering F * v (ii/ii) CABS contraction...
  done

  Using 614.7 KB hard disk for storing "Slater and Yukawa ints (2gamma)"

  Creating Slater (ii/ii) 2gamma  of size 4.1 KB
  done

  Entering F * F (ii/ii) OBS contraction...
  done

  block 0 : 0 : 0
  0.05904  0.00000 -0.00000 -0.00000  0.00000  0.02519 -0.00000 -0.00000 -0.00000 -0.00000  0.02519  0.00000 -0.00000 -0.00000  0.00000  0.02519
  0.00000  0.05611 -0.00000 -0.00000  0.02467  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000
 -0.00000 -0.00000  0.05611  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.02467  0.00000  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000
 -0.00000 -0.00000  0.00000  0.05611 -0.00000  0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.02467 -0.00000  0.00000  0.00000
  0.00000  0.02467 -0.00000 -0.00000  0.05611  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000
  0.02519  0.00000  0.00000  0.00000  0.00000  0.06292 -0.00000 -0.00000  0.00000 -0.00000  0.00675  0.00000  0.00000  0.00000  0.00000  0.00675
 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.05011  0.00000  0.00000  0.00606 -0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000
 -0.00000  0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.00000  0.05011  0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00606 -0.00000 -0.00000
 -0.00000 -0.00000  0.02467  0.00000 -0.00000  0.00000  0.00000  0.00000  0.05611  0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000
 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00606  0.00000  0.00000  0.05011 -0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000
  0.02519  0.00000  0.00000 -0.00000  0.00000  0.00675 -0.00000  0.00000  0.00000 -0.00000  0.06292  0.00000 -0.00000 -0.00000  0.00000  0.00675
  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00000  0.05011  0.00000 -0.00000  0.00606  0.00000
 -0.00000 -0.00000  0.00000  0.02467 -0.00000  0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.05611 -0.00000 -0.00000  0.00000
 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000  0.00606 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.05011 -0.00000  0.00000
  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00606 -0.00000 -0.00000  0.05011  0.00000
  0.02519 -0.00000  0.00000  0.00000 -0.00000  0.00675 -0.00000 -0.00000  0.00000 -0.00000  0.00675  0.00000  0.00000  0.00000  0.00000  0.06292

  Entering F * F (ii/ii) CABS contraction...
  done

  block 0 : 0 : 0
  0.00198 -0.00000  0.00000  0.00000 -0.00000 -0.00018 -0.00000 -0.00000  0.00000 -0.00000 -0.00018  0.00000  0.00000 -0.00000  0.00000 -0.00018
 -0.00000  0.00092  0.00000  0.00000 -0.00030 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000
  0.00000  0.00000  0.00092 -0.00000  0.00000 -0.00000  0.00000  0.00000 -0.00030  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000
  0.00000  0.00000 -0.00000  0.00092 -0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00030  0.00000 -0.00000  0.00000
 -0.00000 -0.00030  0.00000 -0.00000  0.00092 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00000
 -0.00018 -0.00000 -0.00000  0.00000 -0.00000  0.00124 -0.00000  0.00000 -0.00000 -0.00000 -0.00055  0.00000  0.00000  0.00000  0.00000 -0.00055
 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00116  0.00000  0.00000  0.00063  0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000
 -0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00116  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00063 -0.00000 -0.00000
  0.00000  0.00000 -0.00030 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.00092  0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000
 -0.00000  0.00000  0.00000  0.00000  0.00000 -0.00000  0.00063  0.00000  0.00000  0.00116  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000
 -0.00018  0.00000  0.00000 -0.00000  0.00000 -0.00055  0.00000 -0.00000  0.00000  0.00000  0.00124  0.00000 -0.00000 -0.00000  0.00000 -0.00055
  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00116 -0.00000 -0.00000  0.00063 -0.00000
  0.00000 -0.00000 -0.00000 -0.00030  0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00092  0.00000 -0.00000  0.00000
 -0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00063  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00116  0.00000 -0.00000
  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00063 -0.00000  0.00000  0.00116 -0.00000
 -0.00018  0.00000  0.00000  0.00000  0.00000 -0.00055  0.00000 -0.00000  0.00000  0.00000 -0.00055 -0.00000  0.00000 -0.00000 -0.00000  0.00124

  block 0 : 0 : 0
  0.00178 -0.00000 -0.00000 -0.00000 -0.00000  0.00127  0.00000  0.00000 -0.00000  0.00000  0.00127 -0.00000 -0.00000  0.00000 -0.00000  0.00127
 -0.00000  0.00325 -0.00000 -0.00000  0.00189  0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000
 -0.00000 -0.00000  0.00325  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00189 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
 -0.00000 -0.00000  0.00000  0.00325  0.00000 -0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.00189  0.00000  0.00000  0.00000
 -0.00000  0.00189  0.00000  0.00000  0.00325  0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000
  0.00127  0.00000 -0.00000 -0.00000  0.00000  0.00356  0.00000 -0.00000 -0.00000  0.00000  0.00086 -0.00000 -0.00000 -0.00000 -0.00000  0.00086
  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00233  0.00000 -0.00000  0.00037  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000
  0.00000 -0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00233 -0.00000 -0.00000  0.00000  0.00000  0.00000  0.00037  0.00000 -0.00000
 -0.00000  0.00000  0.00189  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00325 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00037 -0.00000 -0.00000  0.00233  0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000
  0.00127 -0.00000 -0.00000  0.00000 -0.00000  0.00086  0.00000  0.00000 -0.00000  0.00000  0.00356  0.00000  0.00000  0.00000  0.00000  0.00086
 -0.00000 -0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00233  0.00000  0.00000  0.00037  0.00000
 -0.00000  0.00000  0.00000  0.00189 -0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00325 -0.00000  0.00000  0.00000
  0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00037 -0.00000  0.00000  0.00000  0.00000 -0.00000  0.00233  0.00000 -0.00000
 -0.00000 -0.00000  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00037  0.00000  0.00000  0.00233  0.00000
  0.00127  0.00000  0.00000  0.00000  0.00000  0.00086  0.00000 -0.00000  0.00000  0.00000  0.00086  0.00000  0.00000 -0.00000  0.00000  0.00356

**** debug ****  X contrib. for debug0.0048279322
**** debug ****  V contrib. -0.1159996248
**** debug ****  X contrib. -0.0106935868
  block 0 : 0 : 0
  0.14131  0.00000 -0.00000 -0.00000  0.00000  0.05911 -0.00000 -0.00000 -0.00000 -0.00000  0.05911  0.00000 -0.00000 -0.00000  0.00000  0.05911
  0.00000  0.13563  0.00000 -0.00000  0.05911  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.00000
 -0.00000  0.00000  0.13563  0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.05911  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
 -0.00000 -0.00000  0.00000  0.13563 -0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.05911  0.00000  0.00000  0.00000
  0.00000  0.05911 -0.00000 -0.00000  0.13563  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000
  0.05911  0.00000 -0.00000 -0.00000  0.00000  0.15238  0.00000 -0.00000 -0.00000  0.00000  0.01589  0.00000 -0.00000 -0.00000  0.00000  0.01589
 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.12061  0.00000  0.00000  0.01589  0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000
 -0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.12061 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.01589  0.00000 -0.00000
 -0.00000 -0.00000  0.05911  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.13563  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.01589  0.00000  0.00000  0.12061  0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000
  0.05911  0.00000 -0.00000  0.00000  0.00000  0.01589  0.00000  0.00000 -0.00000  0.00000  0.15238  0.00000  0.00000  0.00000  0.00000  0.01589
  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.12061  0.00000 -0.00000  0.01589  0.00000
 -0.00000 -0.00000  0.00000  0.05911 -0.00000 -0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.13563  0.00000  0.00000  0.00000
 -0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.01589 -0.00000  0.00000  0.00000 -0.00000  0.00000  0.12061  0.00000 -0.00000
  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000  0.01589  0.00000  0.00000  0.12061  0.00000
  0.05911  0.00000  0.00000  0.00000  0.00000  0.01589  0.00000 -0.00000  0.00000 -0.00000  0.01589  0.00000  0.00000 -0.00000  0.00000  0.15238

**** debug ****  T contrib. 0.2222195422
  Creating Q intermediate: stg2 (OBS) 1/2  of size 4.1 KB
  done

  Creating Q intermediate: stg2, direct (CABS) 2/2  of size 4.1 KB
  done

  block 0 : 0 : 0
  0.00745  0.00000 -0.00000  0.00000  0.00000  0.03572 -0.00000 -0.00000 -0.00000 -0.00000  0.03572  0.00000  0.00000 -0.00000  0.00000  0.03572
  0.00000  0.05281  0.00000  0.00000  0.01246  0.00000 -0.00000 -0.00000 -0.00000 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00000
 -0.00000  0.00000  0.05281  0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.01246  0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
  0.00000  0.00000 -0.00000  0.05281 -0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.01246  0.00000  0.00000  0.00000
  0.00000  0.01246 -0.00000 -0.00000  0.05281  0.00000 -0.00000 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00000
 -0.01079  0.00000 -0.00000 -0.00000  0.00000  0.10655 -0.00000  0.00000 -0.00000 -0.00000  0.00917  0.00000 -0.00000  0.00000  0.00000  0.00917
 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.08821 -0.00000  0.00000  0.00917 -0.00000 -0.00000 -0.00000  0.00000  0.00000  0.00000
 -0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000  0.08821  0.00000  0.00000  0.00000  0.00000  0.00000  0.00917 -0.00000  0.00000
 -0.00000 -0.00000  0.01246  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.05281  0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000
 -0.00000 -0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00917  0.00000  0.00000  0.08821 -0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000
 -0.01079 -0.00000  0.00000  0.00000 -0.00000  0.00917 -0.00000 -0.00000  0.00000 -0.00000  0.10655  0.00000  0.00000 -0.00000  0.00000  0.00917
 -0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000 -0.00000  0.00000  0.00000  0.00000  0.00000  0.08821  0.00000 -0.00000  0.00917  0.00000
  0.00000 -0.00000  0.00000  0.01246  0.00000 -0.00000  0.00000  0.00000 -0.00000  0.00000 -0.00000  0.00000  0.05281  0.00000  0.00000  0.00000
 -0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000  0.00000  0.00917  0.00000 -0.00000  0.00000 -0.00000  0.00000  0.08821  0.00000  0.00000
 -0.00000  0.00000  0.00000  0.00000  0.00000  0.00000  0.00000 -0.00000 -0.00000 -0.00000  0.00000  0.00917  0.00000  0.00000  0.08821  0.00000
 -0.01079  0.00000  0.00000  0.00000  0.00000  0.00917 -0.00000 -0.00000  0.00000 -0.00000  0.00917  0.00000  0.00000 -0.00000  0.00000  0.10655

**** debug ****  Q contrib.: 0.1204101261
  Creating P1: R^PQ_ij K^R_P R^kl_RQ case1, OBS 1/2  of size 50.2 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case1, CABS 2/2  of size 50.2 KB
  done

  Entering P1: R^PQ_ij K^R_P R^kl_RQ case1 contraction...
  done

**** debug ****  P1 case1 contrib.: 0.1430082946
  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, OBS 1/6  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, CABS 2/6  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, OBS 3/6  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, CABS 4/6  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, CABS 5/6  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case2, CABS2 6/6  of size 236.5 KB
  done

  Entering P1: R^PQ_ij K^R_P R^kl_RQ case2 contraction...
  done

**** debug ****  P1 case2 summed: 0.1454487319
  Creating P1: R^PQ_ij K^R_P R^kl_RQ case3, OBS 1/2  of size 236.5 KB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case3, CABS 2/2  of size 236.5 KB
  done

  Entering P1: R^PQ_ij K^R_P R^kl_RQ case3 contraction...
  done

**** debug ****  P1 case3 summed: 0.1474400121
  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, OBS 1/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, CABS 2/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, CABS2 3/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, OBS 4/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, CABS 5/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, CABS 6/7  of size 1.1 MB
  done

  Creating P1: R^PQ_ij K^R_P R^kl_RQ case4, CABS2 7/7  of size 1.1 MB
  done

  Entering P1: R^PQ_ij K^R_P R^kl_RQ case4 contraction...
  done

**** debug ****  P1 case4 summed: 0.1480649224
**** debug ****  P1 contrib.: 0.2961298448
K =  0:
-1.000409 -0.000000  0.002100 -0.000000 -0.000000 -0.000000  0.000000  0.000000  0.000000 -0.000000 -0.000000 -0.000000 -0.000000 -0.000000 
 0.000000  0.517840  0.565216 -0.000000 -0.000000  0.000000  0.000000 -0.000000  0.000000  0.000000 -0.000000 -0.000000  0.000000 -0.000000 
 0.000000  0.000000  0.000000  0.683465  0.018428 -0.132705  0.447150  0.012056 -0.086821  0.000000 -0.000000  0.000000  0.000000 -0.000000 
-0.000000 -0.000000  0.000000  0.100678  0.384524  0.571909  0.065867  0.251571  0.374166 -0.000000 -0.000000  0.000000 -0.000000  0.000000 
 0.000000  0.000000  0.000000 -0.088399  0.580411 -0.374678 -0.057834  0.379728 -0.245130 -0.000000 -0.000000 -0.000000  0.000000  0.000000 
-0.000000 -0.000000  0.000000  0.497041  0.751993 -0.092512 -0.574915 -0.869811  0.107006 -0.000000 -0.000000  0.000000 -0.000000 -0.000000 
 0.000000  0.000000  0.000000 -0.754475  0.501384  0.021968  0.872682 -0.579939 -0.025410  0.000000  0.000000  0.000000  0.000000 -0.000000 
-0.000000 -0.000000  0.000000  0.069419  0.064977  0.901144 -0.080295 -0.075158 -1.042331  0.000000  0.000000  0.000000  0.000000  0.000000 
-0.621576 -1.542649  1.520728 -0.000000 -0.000000 -0.000000  0.000000  0.000000  0.000000 -0.000000 -0.000000  0.000000  0.000000 -0.000000 
-0.000000 -0.000000  0.000000 -0.000000  0.000000  0.000000  0.000000 -0.000000  0.000000 -0.038763  0.088568  0.220096 -0.159323  0.957511 
-0.000000 -0.000000  0.000000  0.000000  0.000000 -0.000000 -0.000000 -0.000000  0.000000  0.986231  0.097208  0.057780  0.114930  0.036776 
 0.000000  0.000000 -0.000000 -0.000000 -0.000000 -0.000000  0.000000  0.000000 -0.000000  0.062000 -0.125897  0.671483 -0.681960 -0.253667 
 0.000000  0.000000 -0.000000  0.000000 -0.000000 -0.000000 -0.000000  0.000000  0.000000 -0.097337 -0.229906  0.669014  0.699777 -0.020018 
 0.000000  0.000000 -0.000000 -0.000000  0.000000 -0.000000 -0.000000 -0.000000  0.000000  0.111930 -0.956033 -0.223045 -0.081551  0.130663 

  Orthogonalizing overlap matrix

  Maximum residual in orthogonalization: 3.797540510454011
  Minimum residual in orthogonalization: 0.001074712569170

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=p, OBS 1/8  of size 17.9 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=p, CABS 2/8  of size 17.9 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=p, OBS 3/8  of size 17.9 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=p, CABS 4/8  of size 17.9 KB
  done

  Entering P2: R^Pm_ij f^Q_P R^kl_Qm (P=p) contraction...
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=A, OBS 5/8  of size 84.5 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=A, CABS 6/8  of size 84.5 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=A, OBS 7/8  of size 84.5 KB
  done

  Creating P2: R^Pm_ij f^Q_P R^kl_Qm, P=A, CABS 8/8  of size 84.5 KB
  done

  Entering P2: R^Pm_ij f^Q_P R^kl_Qm (P=p) contraction...
  done

**** debug ****  P2 contrib.: -0.0637280598
  Creating P3: R^Am_ij f^m_n R^kl_An, 1/2, OBS  of size 84.5 KB
  done

  Creating P3: R^mA_ij f^m_n R^kl_nA, 1/2, CABS  of size 84.5 KB
  done

  Creating P3: R^Am_ij f^m_n R^kl_An, 2/2, OBS  of size 84.5 KB
  done

  Creating P3: R^mA_ij f^m_n R^kl_nA, 2/2, CABS  of size 84.5 KB
  done

  Entering P3: R^Am_ij f^m_n R^kl_An contraction...
  done

**** debug ****  P3 contrib.: -0.0043530171
  Creating P5A: R^mA_ij f^P_m R^kl_PA, 1/4 CABS_CABS  of size 84.5 KB
  done

  Creating P5A: R^mA_ij f^P_m R^kl_PA, 2/4 OBS_OBS  of size 84.5 KB
  done

  Creating P5A: R^mA_ij f^P_m R^kl_PA, 3/4 CABS_OBS  of size 84.5 KB
  done

  Creating P5A: R^mA_ij f^P_m R^kl_PA, 4/4 OBS_CABS  of size 84.5 KB
  done

  Entering P5A: R^mA_ij f^P_m R^kl_PA contraction...
  done

**** debug ****  P5a contrib.: -0.0087021473
  Creating P4: R^pb_ij f^r_p R^kl_rb, 1/2  of size 32.3 KB
  done

  Creating P4: R^pb_ij f^r_p R^kl_rb, 2/2  of size 32.3 KB
  done

  Entering P4: R^pb_ij f^r_p R^ij_rb contraction...
  done

**** debug ****  P4 contrib.: 0.0125538922
  Creating P5B: R^Ab_ij f^p_A R^kl_pb, 1/2, OBS  of size 32.3 KB
  done

  Creating P5B: R^Ab_ij f^p_A R^kl_pb, 2/2, CABS  of size 32.3 KB
  done

  Entering R^Ab_ij f^p_A R^kl_pb contraction...
  done

**** debug ****  P5b contrib.: 0.0007558496
  ------------------------------------------
    R12 contribution  :   -0.1200383911

      >> V*t          :   -0.1159996248
      >> t*(ei+ej)X*t :   -0.0106935868
      >> t*B*t        :    0.1012672717
  ------------------------------------------


  Normal termination.
  --------------------------------------------------

